docs/src/pages/ui/primitives/passwordfield/demo.tsx

```diff
@@ -0,0 +1,257 @@
+              />
+            </form>
+          </Flex>
+        </View>
+      </Example>
+    </View>
+  );
+};
+
+export const LoginFormExample = () => (
+  <Flex as="form" direction="column" gap="1rem">
+    <Heading level={3}>Login</Heading>
+    <TextField label="Username" name="username" autoComplete="username" />
+    <PasswordField
+      label="Password"
+      name="password"
+      autoComplete="current-password"
+    />
+    <Button type="submit" onClick={(e) => e.preventDefault()}>
+      Login
+    </Button>
+  </Flex>
+);
+
+export const SignUpFormExample = () => (
+  <Flex as="form" direction="column" gap="1rem">
+    <Heading level={3}>Sign Up</Heading>
+    <TextField label="Username" name="username" autoComplete="username" />
+    <PasswordField
+      label="Password"
+      name="password"
+      autoComplete="new-password"
+      descriptiveText="Password must be at least 8 characters"
+    />
+    <Button type="submit" onClick={(e) => e.preventDefault()}>
+      Sign Up
+    </Button>
+  </Flex>
+);
+
+export const ChangePasswordFormExample = () => (
+  <Flex as="form" direction="column" gap="1rem">
+    <Heading level={3}>Change Password</Heading>
+    <TextField label="Username" name="username" autoComplete="username" />
+    <PasswordField
+      label="Current password"
+      name="current_password"
+      autoComplete="current-password"
+      descriptiveText="Password must be at least 8 characters"
+    />
+    <PasswordField
+      label="New password"
+      name="new_password"
+      autoComplete="new-password"
+      descriptiveText="Password must be at least 8 characters"
+    />
+    <PasswordField
+      label="Confirm password"
+      name="confirm_password"
+      autoComplete="new-password"
+    />
+    <Button type="submit" onClick={(e) => e.preventDefault()}>
+      Submit
+    </Button>
+  </Flex>
+);
+
+export const DescriptiveTextExample = () => (
+  <PasswordField
+    label="Password"
+    name="password"
+    descriptiveText={
+      <Text
+        as="span"
+        color="rebeccapurple"
+        fontStyle="italic"
+        fontSize="0.8rem"
+      >
+        Password length must be greater than 8 characters
+      </Text>
+    }
+  />
+);
+
+export const RequiredPasswordFieldExample = () => (
+  <Flex as="form" direction="column" width="20rem">
+    <PasswordField
+      label={
+        <Text>
+          Password
+          <Text as="span" fontSize="0.8rem" color="red">
+            {' '}
+            (required)
+          </Text>
+        </Text>
+      }
+      name="password"
+      isRequired={true}
+    />
+    <PasswordField
+      label="Password"
+      descriptiveText={
+        <Text as="span" fontSize="0.8rem" color="red" fontStyle="italic">
+          Required
+        </Text>
+      }
+      name="password"
+      isRequired={true}
+    />
+    <Button type="submit">Submit</Button>
+  </Flex>
+);
+
+export const PasswordFieldStyledPropsExample = () => (
+  <TextField
```

Comment:
Should this be `<PasswordField`?

Reply:
Whoops, how did I leave that behind? 🤦‍♂️